src/server/src/main/java/io/cassandrareaper/service/ClusterRepairScheduler.java

```diff
@@ -57,6 +57,10 @@ public ClusterRepairScheduler(AppContext context) {
   }
 
   public void scheduleRepairs(Cluster cluster) throws ReaperException {
+    if (context.config.getAutoScheduling().getExcludedClusters().contains(cluster.getName())) {
```

Comment:
We slightly modify the cluster names before storing them, with this : https://github.com/thelastpickle/cassandra-reaper/blob/master/src/server/src/main/java/io/cassandrareaper/core/Cluster.java#L78-L81

This may surprise users trying to exclude clusters from the auto scheduling. You might want to process your list of excluded clusters with this method before checking for the presence of the cluster passed as argument.